rails/etsy_product_catalog/app/models/category.rb

```diff
@@ -0,0 +1,12 @@
+class Category < ActiveRecord::Base
+	#has_many :listings
+
+	def to_title
+		name.gsub("_", " ").titleize.gsub("And", "and")
+	end
```

Comment:
I like this. The method name is clear about what it does, and you can use that method name in a view template while hiding away all the messy implementation details down here. Good stuff.